SOC/models/forest.py

```diff
@@ -66,12 +56,20 @@ def topple_dissipate(self):
 
         self.values, self.new_values = self.new_values, self.values
         self.new_values[...] = 0
-        number_burning = (self.values[self.BC:-self.BC, self.BC:-self.BC] == _burning).sum()
+        number_burning = (self.inside(self.values) == _burning).sum()
         return number_burning
 
 _neighbours = ((-1,-1), (-1,0), (-1,1), (0,-1), (0, 1), (1,-1), (1,0), (1,1))
 @numba.njit
-def burn_trees(new_values, values, f, BC):
+def burn_trees(new_values: np.ndarray, values: np.ndarray, f: float, BC: int):
```

Comment:
[The roof, the roof, the roof is on fire
We don't need no water let the motherfucker burn
Burn motherfucker burn
](https://www.youtube.com/watch?v=ChmUC0OysoU) 😈

Reply:
> [The roof, the roof, the roof is on fire We don't need no water let the motherfucker burn Burn motherfucker burn ](https://www.youtube.com/watch?v=ChmUC0OysoU) smiling_imp

Motto projektu, zaraz pod abstract'em.

Reply:
![las?](https://cdn.vox-cdn.com/thumbor/zL48ecvX2NkW1cU0FEfgrCc7Rgo=/0x0:900x500/920x613/filters:focal(378x178:522x322):format(webp)/cdn.vox-cdn.com/uploads/chorus_image/image/49493993/this-is-fine.0.jpg)